snuba/util.py

```diff
@@ -525,16 +526,17 @@ def raw_query(request: Request, sql, client, timer, stats=None):
             'status': status,
         })
 
-        # send to datadog
-        tags = [
-            'status:{}'.format(status),
-            'referrer:{}'.format(stats.get('referrer', 'none')),
-            'final:{}'.format(stats.get('final', False))
-        ]
-        mark_tags = [
-            'final:{}'.format(stats.get('final', False))
-        ]
-        timer.send_metrics_to(metrics, tags=tags, mark_tags=mark_tags)
+        timer.send_metrics_to(
```

Comment:
note: Not covered.

Reply:
This use still isn't covered (surprisingly) but the implementation of `send_metrics_to` is now covered.

Reply:
This is not covered because it dependson RECORD_QUERIES which is false in dev and tests

Reply:
I think that it is wider in scope than this PR, but we should look into a way to execute the query recording path in development (at least up until the point of actually publishing the message.)